### PR TITLE
LRQA-82910 | Test Fix for flaky testcase

### PR DESCRIPTION
**Pull request by Hijky**
Background:
Test Fix for flaky testcase FilterSortAndSearchTheListOfAPIApplications#CanSortAPIsByStatus

Jira ticket:
https://liferay.atlassian.net/browse/LRQA-82910

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Review by yanan — APPROVED** [review]

File: modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/FilterSortAndSearchTheListOfAPIApplications.testcase [hunk 1]
```diff
@@ -184,7 +184,7 @@ definition {
 		}
 
 		task ("Then the APIs are sorted by Status") {
-			APIBuilderUI.viewItemsInOrder(itemList = "test1,test2,test3");
+			APIBuilderUI.viewItemsInOrder(itemList = "test1");
```
Comment (magjed4289) [inline modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/FilterSortAndSearchTheListOfAPIApplications.testcase:187]: shouldn't we see test1,test3,test2?

Comment (Hijky) [reply modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/FilterSortAndSearchTheListOfAPIApplications.testcase:187]: Actually, the results are not stable, sometimes test1,test2,test3, and sometimes test1,test3,test2, it's the test failed reasons. The dev said it has been sorted by status and it's not a bug, so I talked with Ashley, and we think the best way to solve the flaky is we only assert the first published application

Comment (magjed4289) [reply modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/FilterSortAndSearchTheListOfAPIApplications.testcase:187]: ok, well we're testing that it's being sorted by status, so why are not we checking the Status field? We should check whether it's PUBLISHED,PUBLISHED,UNPUBLISHED

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Review by magjed4289 — APPROVED** [review]

**Review by yanan — APPROVED** [review]

**Comment by Hijky** [comment]
All the failures are about 404 errors with API Builder, it also has a [ticket](https://liferay.atlassian.net/browse/LPS-193414) and has been merged.

**Comment by Hijky** [comment]
ci:forward

**Pull request closed without merging**